### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by kennyweiss**
Includes links to known user projects.

Also adds a `mailmap` file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,12 +4,21 @@
 [![Build Status](https://ci.appveyor.com/api/projects/status/fuaftu9mvp0y488j/branch/master?svg=true)](https://ci.appveyor.com/project/cyrush/blt/branch/master)
 [![Documentation Status](https://readthedocs.org/projects/llnl-blt/badge/?version=latest)](https://llnl-blt.readthedocs.io/en/latest/?badge=latest)
 
-A streamlined CMake build system foundation for developing HPC software
+BLT is a streamlined `CMake <https://cmake.org/>`_-based foundation for building, linking and testing HPC software.
 
-User Tutorial
----------------------
+BLT makes is easy to get up and running on a wide range of compilers (gcc, clang, intel, XL, Visual Studio),
```
Comment (kennyweiss) [inline README.md]: I wanted to add a bit of context here about `blt` and what it is for.

Please update/improve/work on this more.

File: README.md [hunk 2]
```diff
@@ -31,19 +40,36 @@ Developers include:
...
+Projects using this library
+---------------------------
+
+ * `Ascent <https://github.com/Alpine-DAV/ascent>`_: A flyweight in-situ visualization and analysis runtime for multi-physics HPC simulations
+ * `Axom <https://github.com/LLNL/axom>`_: Software infrastructure for the development of multi-physics applications and computational tools
+ * `CHAI <https://github.com/LLNL/CHAI>`_: Copy-hiding array abstraction to automatically migrate data between memory spaces
+ * `Conduit <https://github.com/LLNL/conduit>`_: Simplified data exchange for HPC simulations
+ * `RAJA <https://github.com/LLNL/raja>`_: Performance portability layer for HPC
+ * `Umpire <https://github.com/LLNL/Umpire>`_: Application-focused API for memory management on NUMA and GPU architectures
+ * `VTK-h <https://github.com/Alpine-DAV/vtk-h>`_: Scientific visualization algorithms for emerging processor architectures
```
Comment (kennyweiss) [inline README.md]: Looks like github's markdown is not happy with these links.  

- [ ] Fix this.

File: .mailmap [hunk 3]
```diff
@@ -0,0 +1,10 @@
+George Zagaris <zagaris2@llnl.gov>         George Zagaris <george.zagaris@gmail.com>
+Kenneth Weiss <kweiss@llnl.gov>            Kenneth Weiss <weiss27@llnl.gov>
+Kenneth Weiss <kweiss@llnl.gov>            Kenny Weiss <kenny@kennyweiss.com>
+Kenneth Weiss <kweiss@llnl.gov>            Kenny Weiss <kennyweiss@users.noreply.github.com>
+Peter B. Robinson <robinson96@llnl.gov>    robinson96 <robinson96@llnl.gov>
+Peter B. Robinson <robinson96@llnl.gov>    robinspb <manbearkitty@gmail.com>
+Randolph R. Settgast <settgast1@llnl.gov>  Randolph R. Settgast <rrsettgast@gmail.com>
+Randolph R. Settgast <settgast1@llnl.gov>  Randolph Settgast <rrsettgast@gmail.com>
+Randolph R. Settgast <settgast1@llnl.gov>  Randolph Settgast <settgast1@llnl.gov>
+
```
Comment (kennyweiss) [inline .mailmap]: Please let me know if you'd like a different mapping.

File: README.md [hunk 4]
```diff
@@ -4,12 +4,22 @@
 [![Build Status](https://ci.appveyor.com/api/projects/status/fuaftu9mvp0y488j/branch/master?svg=true)](https://ci.appveyor.com/project/cyrush/blt/branch/master)
 [![Documentation Status](https://readthedocs.org/projects/llnl-blt/badge/?version=latest)](https://llnl-blt.readthedocs.io/en/latest/?badge=latest)
 
-A streamlined CMake build system foundation for developing HPC software
+BLT is a streamlined [CMake](https://cmake.org)-based foundation for
```
Comment (gzagaris) [inline README.md]: How about something like this:

> BLT is a [CMake](https://cmake.org)-based build system framework designed to streamline the development and simplify the maintenance of build systems for large-scale HPC applications.

or

> BLT is a [CMake](https://cmake.org)-based build system framework designed to streamline <b>B</b>uilding, <b>L</b>inking and <b>T</b>esting large-scale HPC applications.

File: README.md [hunk 5]
```diff
@@ -4,12 +4,22 @@
 [![Build Status](https://ci.appveyor.com/api/projects/status/fuaftu9mvp0y488j/branch/master?svg=true)](https://ci.appveyor.com/project/cyrush/blt/branch/master)
 [![Documentation Status](https://readthedocs.org/projects/llnl-blt/badge/?version=latest)](https://llnl-blt.readthedocs.io/en/latest/?badge=latest)
 
-A streamlined CMake build system foundation for developing HPC software
+BLT is a streamlined [CMake](https://cmake.org)-based foundation for 
+<b>b</b>uilding, <b>l</b>inking and <b>t</b>esting HPC software.
 
-User Tutorial
----------------------
+BLT makes is easy to get up and running on a wide range of compilers (gcc, clang, intel, XL, Visual Studio),
```
Comment (gzagaris) [inline README.md]: Minor typo: `BLT makes it easy....`

File: README.md [hunk 6]
```diff
@@ -4,12 +4,22 @@
...
-User Tutorial
----------------------
+BLT makes is easy to get up and running on a wide range of compilers (gcc, clang, intel, XL, Visual Studio), 
+operating systems (Linux, Mac, Windows) and HPC technologies (MPI, OpenMP, CUDA, HIP).
 
-https://llnl-blt.readthedocs.io/en/latest/
+Getting started
+---------------
+
+BLT is easy to pull into in your existing or new CMake-based project using a single CMake `include()` command:
```
Comment (gzagaris) [inline README.md]: Typo: scratch `in`

I would rephrase it slightly as:

> BLT is easy to pull into an existing or new CMake-based project...

File: README.md (hunk 1 above)
Comment (white238) [reply README.md]: maybe a bulleted list for each instead of parenthesis list?

* Compilers: gcc, clang, intel,...
* Operating Systems: OSX, Linux, Windows

Comment (kennyweiss) [reply README.md]: Thanks @white238 -- do you want to add some content/wording from your recent slides?

File: README.md (hunk 4 above)
Comment (kennyweiss) [reply README.md]: Thanks for the suggestions @gzagaris -- I'll update them later today.

File: README.md (hunk 1 above)
Comment (gzagaris) [reply README.md]: I also think bullets would read a lot easier.

File: README.md (hunk 4 above)
Comment (gzagaris) [reply README.md]: Thanks!

File: README.md (hunk 1 above)
Comment (white238) [reply README.md]: Looks fine to me. Thanks kenny

Comment (kennyweiss) [reply README.md]: ok -- If we're going to bullets, I'll also add a line about built-in testing, benchmarking, documentation and code health frameworks.

File: README.md (hunk 4 above)
Comment (gzagaris) [reply README.md]: Did you intend to make this change?

Comment (kennyweiss) [reply README.md]: Thanks @gzagaris -- I made the change
